src/script/components/Conversation/Conversation.tsx

```diff
@@ -540,7 +538,7 @@ export const Conversation = ({
           />
 
           {isConversationLoaded &&
-            (isReadOnlyConversation || (!isTemporaryGuest && !isActiveParticipant) ? (
+            (isReadOnlyConversation || isSelfUserRemoved ? (
```

Comment:
It would make more sense not to render neither inputbar nor readonly message (so `isConversationLoaded && !isSelfUserRemoved`), I don't think we have a readonly message for this case, do we?